### PR TITLE
[IMP] marketing: workflow_activities.rst - Format fixes

### PR DESCRIPTION
**Pull request by ksc-odoo**
added necessary RST formatting adjustments to adhere to decisions made during recent Writer's Workshop meeting

**This PR should be FWP to 15.0 & 16.0**

Project Task: https://www.odoo.com/web#id=3329146&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

File: content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst [hunk 1]
```diff
@@ -2,13 +2,16 @@
...
-When creating a campaign in the :guilabel:`Marketing Automation` app, users can plan marketing
-activities such as email or SMS campaigns. To get started, navigate to the :guilabel:`Workflow`
-area, and click :guilabel:`Add New Activity`. A blank activity template will appear where
-parameters can be set for that specific activity.
+When creating a campaign in the *Marketing Automation* app, users can plan marketing activities,
+such as email or SMS campaigns.
 
-When :guilabel:`Add New Activity` is selected, a blank activity template will appear with the
-following customizable fields:
+To get started, navigate to the bottom of a marketing automation campaign detail form,  and click
```
Comment (samueljlieber) [inline content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst:8]:
```suggestion
To get started, navigate to the bottom of a marketing automation campaign detail form, and click
```

File: content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst [hunk 2]
```diff
@@ -27,40 +30,46 @@ following customizable fields:
...
    been :guilabel:`Sent`, and what percentage of those have been :guilabel:`Clicked`,
    :guilabel:`Replied` to, or :guilabel:`Bounced`.
 
 Child activities
 ================
 
-Odoo also has the option to :guilabel:`Add Child Activity`. *Child Activities* are sub-activities
-that are connected to (and triggered by) the activity above it, which is also known as its *Parent
-Activity*.
+There is also the option to add a *Child Activity* by clicking :guilabel:`Add child activity`,
```
Comment (samueljlieber) [inline content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst:63]: No need to capitalize _child activity_
```suggestion
There is also the option to add a *child activity* by clicking :guilabel:`Add child activity`,
```

File: content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst [hunk 3]
```diff
@@ -27,40 +30,46 @@ following customizable fields:
...
+located at the bottom of each activity block in the :guilabel:`Workflow` section of a marketing
+automation form.
+
+*Child Activities* are sub-activities that are connected to (and triggered by) the activity above
+it, which is also known as its *Parent Activity*.
```
Comment (samueljlieber) [inline content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst:68]:
```suggestion
Child activities are sub-activities that are connected to (and triggered by) the activity above it,
which is also known as its *parent activity*.
```

File: content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst [hunk 4]
```diff
@@ -77,5 +86,6 @@ activity, hover over :guilabel:`Add child activity`, and select the trigger:
 - :guilabel:`Bounced`: the next activity will be triggered if the mailing is bounced (not sent).
 
 Once a trigger is selected, the user can configure the child activity (it has the same
-configuration options as a regular activity) and click :guilabel:`Save & Close`` to finish creating
-the child activity.
+configuration options as a regular activity), and click :guilabel:`Save & Close` to finish creating
+the child activity, which will then be displayed in the :guilabel:`Workflow` section, in a slightly
+indented position beneath its parent activity.
```
Comment (samueljlieber) [inline content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst:91]:
```suggestion
Once a trigger is selected, the user can configure the child activity (it has the same configuration
options as a regular activity), and click :guilabel:`Save & Close` to finish creating the child
activity, which will then be displayed in the :guilabel:`Workflow` section, in a slightly indented
position beneath its parent activity.
```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @ksc-odoo please see my technical changes below and let me know if you approve or have any changes 🙂 I will push these changes up after this review.

**Comment by samueljlieber** [comment]
Implemented my [technical changes](https://github.com/odoo/documentation/pull/4322#pullrequestreview-1411925297) in https://github.com/odoo/documentation/pull/4322#pullrequestreview-1411925297.

**Comment by ksc-odoo** [comment]
> Hi @ksc-odoo please see my technical changes below and let me know if you approve or have any changes 🙂 I will push these changes up after this review.

@samueljlieber everything looks good to me - thanks again for the always-awesome reviews!

**Comment by samueljlieber** [comment]
Hi @StraubCreative this PR is ready for your review!

**Review by StraubCreative — APPROVED** [review]
Seems good. Thanks all 👍

**Pull request closed without merging**